nxp_ls1028ardb-64b_ubuntu_defconfig build error

Does "nxp_ls1028ardb-64b_ubuntu_defconfig" supported ?
It resulted in the following error, even though the system has the debootstrap package.

OS is Debian 10.9.
I'm using "OpenIL-v1.11-202104" tag.

```
$make nxp_ls1028ardb-64b_ubuntu_defconfig
$make
*************************
debootstrap not found, Try install debootstrap package.
*************************
make: *** [Makefile:1045: _check_ubuntu_env] Error 1
$dpkg -l |grep debootstrap
ii  debootstrap                           1.0.114                                      all          Bootstrap a basic Debian system
```

Thanks.

--- a/env_setup.sh
+++ b/env_setup.sh
@@ -5,7 +5,7 @@ host_packages_list="vim git make net-tools build-essential cpio autotools-dev au
 main()
 {
 	echo "Checking the packages required on host!"
-        echo "These packages will be installed if they are not exist!"
+        echo "These packages will be installed if they do not exist!"
 
 	if ! dpkg-query -l debootstrap | grep ii 1>/dev/null; then
 		echo "Installing debootstrap-1.0.95ubuntu0.10"

--- a/env_setup.sh
+++ b/env_setup.sh
@@ -8,19 +8,19 @@ main()
         echo "These packages will be installed if they are not exist!"
 
 	if ! dpkg-query -l debootstrap | grep ii 1>/dev/null; then
-		echo "Installing debootstrap-1.0.95ubuntu0.8"
-		wget http://ports.ubuntu.com/pool/main/d/debootstrap/debootstrap_1.0.95ubuntu0.8_all.deb
-		sudo dpkg -i debootstrap_1.0.95ubuntu0.8_all.deb
-		rm debootstrap_1.0.95ubuntu0.8_all.deb
+		echo "Installing debootstrap-1.0.95ubuntu0.10"
+		wget http://ports.ubuntu.com/pool/main/d/debootstrap/debootstrap_1.0.95ubuntu0.10_all.deb
+		sudo dpkg -i debootstrap_1.0.95ubuntu0.10_all.deb
+		rm debootstrap_1.0.95ubuntu0.10_all.deb
 	else
-		if ! dpkg-query -l debootstrap | grep 1.0.95ubuntu0.8 1>/dev/null; then
-			echo "Reinstalling debootstrap (1.0.95ubuntu0.8)"
+		if ! dpkg-query -l debootstrap | grep 1.0.95ubuntu0.10 1>/dev/null; then
+			echo "Reinstalling debootstrap (1.0.95ubuntu0.10)"
 			sudo apt remove -y debootstrap
-			wget http://ports.ubuntu.com/pool/main/d/debootstrap/debootstrap_1.0.95ubuntu0.8_all.deb
-			sudo dpkg -i debootstrap_1.0.95ubuntu0.8_all.deb
-			rm debootstrap_1.0.95ubuntu0.8_all.deb
+			wget http://ports.ubuntu.com/pool/main/d/debootstrap/debootstrap_1.0.95ubuntu0.10_all.deb
+			sudo dpkg -i debootstrap_1.0.95ubuntu0.10_all.deb
+			rm debootstrap_1.0.95ubuntu0.10_all.deb
 		else
-			echo "debootstrap-1.0.95ubuntu0.8 is ready"
+			echo "debootstrap-1.0.95ubuntu0.10 is ready"
 		fi
 	fi
 

